### PR TITLE
chore(beta-components): promote 2022.14 candidates

### LINKED ISSUE #5178: Promote beta components
Remove beta tags:
- Calendar month
- Card: raised variant
- Code editor
- Dual list selector
- Form group: icon sprites
- Label group (except editable variant)
- Masthead
- Menu toggle
- File upload - multiple
- Notification drawer
- Panel
- Select: view more, loading variant
- Slider
- Tree view

### PR DESCRIPTION
**Pull request by nicolethoen**
Closes #5178

Remove beta tags:

Calendar month
Card: raised variant
Code editor
Dual list selector
Form group: icon sprites
Label group (except editable variant)
Masthead
Menu toggle
File upload - multiple
Notification drawer
Panel
Select: view more, loading variant
Slider
Tree view

### CODE REVIEW

**Review by mcarrano — CHANGES_REQUESTED** [review]
Looks good @nicolethoen except for Code Editor. Looks like you missed that one.

**Comment by nicolethoen** [comment]
good catch!

**Review by mcarrano — APPROVED** [review]
Looks good. Thanks @nicolethoen !

File: src/patternfly/components/NotificationDrawer/examples/NotificationDrawer.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ---
 id: Notification drawer
 section: components
-beta: true
```
Comment (mcoker) [inline src/patternfly/components/NotificationDrawer/examples/NotificationDrawer.md:4]: 🥳

**Review by mcoker — APPROVED** [review]
Nice!

**Pull request merged**